imgp

```diff
@@ -275,6 +277,10 @@ def resize_image(src):
 
         if convert and _format != 'PNG':
             return
+
```

Comment:
No unnecessary newlines for each if-else please. At least look at the existing code.